### PR TITLE
Implement API route To return a specific "Experience"

### LINKED ISSUE #20: Implement API route to return a specific Experience
On the `/resume/experience` route, you should

- [ ] Return a specific Experience when a GET request is made with an index as an input. The Experience should be in a JSON format.

### PR DESCRIPTION
**Pull request by davidkurilla**
Closes #20 

I tested the API route manually with curl.

### CODE REVIEW

**Comment by davidkurilla** [comment]
@Pradyuman7

File: app.py [hunk 1]
```diff
@@ -52,6 +52,15 @@ def experience():
...
     return jsonify({})
 
+@app.route('/resume/experience/<int:index>', methods=['GET'])
+def experience_by_index(index):
+    '''
+    Handle experience requests by index
+    '''
+    if 0 <= len(data['experience']) and index < len(data['experience']):
+        return jsonify(data['experience'][index])
+    return jsonify({'error': 'Index out of range'}), 404
```
Comment (Pradyuman7) [inline app.py:62]: nit: Let's say experience not found to make it a bit more generic and not give away the implementation detail of checking by index

Comment (davidkurilla) [reply app.py]: I made the change to the error message as requested.

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💪 let's merge once the conflicts are resolved

**Comment by davidkurilla** [comment]
I believed I have corrected the merge conflict.

@Pradyuman7

**Pull request merged**